### ISSUE TITLE
Efficiency request

### ISSUE THREAD

**Issue by Lucas559-noob** [zymex22/Project-RimFactory-Revived#441]
I was doing some automation, but I noticed that with make beer x50, it take a lot of materials, but since they will keep in loop, only at borders will get it(since the math of like 5 tiles), so there is a way to make it have like a timer to block conveyor, so the material can merge to be 1 thing
![screenshot7](https://user-images.githubusercontent.com/54007604/115981165-4f447380-a568-11eb-8211-a5b512c7673d.png)
example of funny thing
ik i can make the conveyor go around each thing like this
![Capturar](https://user-images.githubusercontent.com/54007604/115981268-e0b3e580-a568-11eb-9f1d-cdb19029474f.PNG)
subtitles: Red: Conveyor   Blue: Underground Conveyor   Black: Fermenters

**Comment by lilwhitemouse** [comment]
So....if there were an option to set conveyors to "Only push full stack"?

**Comment by lilwhitemouse** [comment]
I *think* that might suffice for what you need, and it would be somewhat easy to add.

**Comment by Lucas559-noob** [comment]
> So....if there were an option to set conveyors to "Only push full stack"?

yeah, is a good idea, but i think mods that change stack limit can break it, so it would be better if there is a setting in puller to select a number to pull

**Comment by Sn1p3rr3c0n** [comment]
> How would stack limit changes break it?
if you have a stack limit of 500 - 1000 the issue should be clear.

I realize that i missed certain cases that are important for the logic system to be complete.
I will check if this is something that i want to included the system or if it should be separate.

**Comment by lilwhitemouse** [comment]
Ah, yes, of course.

There is currently no UI that specifies X of item Y; I will want something like that for Deep Storage as well.

The full-stack option might still be useful.
